__tests__/app.test.js

```diff
@@ -88,6 +82,28 @@ describe("GET /api/articles/:article_id", () => {
         expect(response.body.status).toBe(400);
         expect(response.body.msg).toBe("bad request - invalid input type");
       });
+  });
+});
 
+describe("GET /api/users", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
```

Comment:
I think you might need to refactor this, your users should be sent back as an obj with a key of `users` so you might want to take another look at how you'er sending this back from your controller